techniques/system/inventory/1.0/fusionAgent.st

```diff
@@ -141,6 +141,10 @@ bundle agent fusionAgent
     any::
       "inventory_name"           string => "${sys.uqhost}-${g.uuid}.ocs";
       "inventory_path"           string => "${g.rudder_var_tmp}/inventory/${inventory_name}";
+    verbose_inventory::
```

Comment:
why not simply `verbose`

Reply:
this allows running inventory verbosely without having a ton of verbose agent messages